crispy_forms/templates/bootstrap/layout/radioselect.html

```diff
@@ -5,7 +5,7 @@
     {% include 'bootstrap/layout/field_errors_block.html' %}
 
     {% for choice in field.field.choices %}
-        <label class="radio{% if inline_class %} {{ inline_class }}{% endif %}">
+        <label for="id_{{ field.html_name }}_{{ forloop.counter }}" class="radio{% if inline_class %} {{ inline_class }}{% endif %}">
```

Comment:
It sounds like this isn't necessary if the `input` is within the label:

> A label element can have both a for attribute and a contained control element, as long as the for attribute points to the contained control element.

[`for` `label` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#Attributes)